docker-compose.yml

```diff
@@ -16,6 +17,13 @@ services:
             placement:
                 constraints:
                     - node.role == worker
+            update_config:
+              parallelism: 1
+              delay: 10s
+              failure_action: rollback
+              monitor: 10s
+        healthcheck:
+          test: ["CMD", "curl", "-f", "-X", "POST", "-d", "json={\"json\":\"healthcheck\"}", "http://localhost:8888/healthcheck.test"]
```

Comment:
Doesn't this just add a log entry with the data "healthcheck" to the logs? I don't think this is how we want to check the health of the service. Take a look at https://docs.fluentd.org/v1.0/articles/monitoring-prometheus as one example we could use that would also allow us to monitor with prometheus.